src/test/resources/tpcds/spark-2.4/approved-plans-v1_4/q0/explain.txt

```diff
@@ -0,0 +1,11 @@
+== Physical Plan ==
+CollectLimit 100
++- *(2) Project [d_year#1, ss_customer_sk#2]
+   +- *(2) BroadcastHashJoin [d_date_sk#3], [ss_sold_date_sk#4], Inner, BuildRight
```

Comment:
Could you explain why only this file has BroadcastHashJoin?

Reply:
spark defaults tests don't set broadcast threshold to -1. Hyperspace tests set it to -1.